### DISCUSSION TITLE
New containers created by wg-easy? Am I being hacked?

### DISCUSSION THREAD

**Discussion by alvarokrn** [wg-easy/wg-easy#1469] [Q&A] [answered]
Today I realized that I have two new containers created apparently by wg-easy but I searched info about the names (`goofy_stonebraker` and `zealous_northcutt`) and nothing found about it.

![Captura desde 2024-10-13 12-28-45](https://github.com/user-attachments/assets/f5318582-86a5-432b-ad08-779c94506306)

If I watch the logs, they only have something like: PASSWORD_HASH='[a random hash]'.

So I don't know if this is normal or maybe I'm being hacked or something.

**Comment by Deceased-prince** [comment]
It looks like they were created around the same time you created yours. where they running when you found it?

**Reply by alvarokrn** [reply]
They were not running when I found it. They were stopped like they are in the image.

**Reply by Deceased-prince** [reply]
Then odds are probably fine they were probably created by mistake I wouldn't really worry about it if they weren't running but just keep an eye out for anything else

**Comment by shalafi99** [comment, marked as answer]
I had this behavior happening with my Docker environment today and figured out the reason:
The strange/unexpected for container was a byproduct of running `docker run -it ghcr.io/wg-easy/wg-easy wgpw YOUR_PASSWORD`

OP's comment about the container log only having a single line like `PASSWORD_HASH='[a random hash]'` did not strike me about what was going on.

@kaaax0815 a way to fix that behavior would be changing the corresponding docker run command to:
`docker run --rm -it ghcr.io/wg-easy/wg-easy wgpw YOUR_PASSWORD`
I can submit a PR to update the docs, if it's ok with you.

**Reply by kaaax0815** [reply]
fix the issue with the comment from here too https://github.com/wg-easy/wg-easy/issues/1433

I will merge it even with #1482 

But its just temporary as it will be fixed altogether with 15.0.0

**Reply by shalafi99** [reply]
Sure thing, I will cover them both.
Thanks, I was not sure if I could just go ahead and mention that other one here.

**Reply by shalafi99** [reply]
PR #1529
